joomla/joomla-cms
# [4.0] Fix module header sizing

Pull Request for Issue #17814 .

### Testing Instructions
Add varying content to modules in the same position (top-a). Enable titles. Check frontend and ensure title heights are equal.


### Expected result
![modhead2](https://user-images.githubusercontent.com/2803503/29964178-261f4434-8f01-11e7-9a34-d7810de4c7b4.png)



### Actual result
![modhead1](https://user-images.githubusercontent.com/2803503/29964182-28684e7a-8f01-11e7-9022-24e3f2b22dae.png)



### Documentation Changes Required
